wolfcrypt/src/sp_int.c

```diff
@@ -8157,8 +8157,8 @@ int sp_div(const sp_int* a, const sp_int* d, sp_int* r, sp_int* rem)
     sp_int* tr = NULL;
     sp_int* trial = NULL;
 #ifdef WOLFSSL_SP_INT_NEGATIVE
-    int signA = MP_ZPOS;
-    int signD = MP_ZPOS;
+    word32 signA = MP_ZPOS;
```

Comment:
Change to unsigned int. Matches declaration in header file for sign.
(And two other instances.)

Reply:
:heavy_check_mark: